OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -62,43 +68,45 @@ public override Activity Tick(Actor self)
 			if (!closestHarvestablePosition.HasValue)
 			{
 				if (!harv.IsEmpty)
-					return new DeliverResources(self);
+				{
+					return NextActivity;
+				}
```

Comment:
Style nit: We usually don't put brackets around singe-line statements unless compiler or readability require it (which isn't the case here).

Reply:
fixed